### PR TITLE
document configuring service concurrency

### PR DESCRIPTION
**Pull request by senyosimpson**
### Summary of changes

Add documentation on configuring Fly.io concurrency limits when using K8s services

### CODE REVIEW

**Comment by senyosimpson** [comment]
@jipperinbham mind just sanity checking the example

**Review by jipperinbham — APPROVED** [review]

File: kubernetes/services.html.markerb [hunk 1]
```diff
@@ -69,6 +69,31 @@ spec:
       targetPort: 8080
 ```
 
+## Concurrency limits
+
+[Concurrency limits](https://fly.io/docs/reference/configuration/#services-concurrency) are used to limit the load on your application.
```
Comment (andie787) [inline kubernetes/services.html.markerb:74]: In this case, with a cluster having one Machines, there's no load balancing and no auto stop or start, right?

**Review by andie787 — COMMENTED** [review]
lgtm! left one question

File: kubernetes/services.html.markerb (hunk 1 above)
Comment (senyosimpson) [reply kubernetes/services.html.markerb:74]: yep! but if your one instance hits its hard limit, we'll just fail requests to your app entirely

**Pull request merged**